src/Build/BackEnd/Client/MSBuildClient.cs

```diff
@@ -353,6 +363,7 @@ private void ConfigureAndQueryConsoleProperties()
                     IntPtr stdOut = NativeMethodsShared.GetStdHandle(NativeMethodsShared.STD_OUTPUT_HANDLE);
                     if (NativeMethodsShared.GetConsoleMode(stdOut, out uint consoleMode))
                     {
+                        _originalConsoleMode = consoleMode;
```

Comment:
If we moved this line before line `consoleMode |= NativeMethodsShared.ENABLE_VIRTUAL_TERMINAL_PROCESSING | NativeMethodsShared.DISABLE_NEWLINE_AUTO_RETURN;` nr 376 we would revert it to original only when we have had actually changed it. We most probably change it all the time, but still :-)